app/2.0/docs/devguide/encapsulated-style.md

```diff
@@ -0,0 +1,639 @@
+
+You can use CSS selectors to determine when and how to style the host. In this code sample:
+
+* The rule `:host { font-family: sans-serif; }` always matches `<x-foo>` elements
+* The rule `:host(.blue) {color: blue;}` matches `<x-foo>` elements of class `blue`
+* The rule `:host(.red) {color: red;}` matches `<x-foo>` elements of class `red`
+* The rule `:host(:hover)` matches `<x-foo>` elements when they are hovered over
+
+[See it on Plunker](http://plnkr.co/edit/FsXnCAz65SR6fZ7YKuy6?p=preview)
+
+```html
+<dom-module id="x-foo">
+	<template>
+		<style>
+			:host { font-family: sans-serif; }
+			:host(.blue) {color: blue;}
+			:host(.red) {color: red;}
+			:host(:hover) {color: green;}
+		</style>
+		<p>Hi, from x-foo!</p>
+	</template>
+</dom-module>
+<x-foo class="blue"></x-foo>
+<x-foo class="red"></x-foo>
+```
+
+Descendant selectors after `:host` match elements in the shadow tree. In this example, the CSS selector applies to any `p` element in the shadow tree if the host has class "warning":
+
+[See it on Plunker](http://plnkr.co/edit/MRN9blKg6A3w8G0RkyJD?p=preview)
+
+```html
+<dom-module id="x-foo">
+	<template>
+		<style>
+		:host(.warning) p {
+			color:red;
+		}
+		</style>
+    <p>Make this text red if x-foo has class "warning", and black otherwise.</p>
+	</template>
+</dom-module>
+<x-foo class="warning"></x-foo>
+<x-foo></x-foo>
+```
+
+Styling with the `:host` selector is one of two instances where rules inside a shadow tree can affect an element outside a shadow tree. The second instance uses the `::slotted()` syntax to apply styling rules to content populated from the main DOM at runtime.
+
+### Styling slotted content (distributed children)
+
+You can create slots in an element's template that are populated at runtime using this syntax:
+
+[See it on Plunker](http://plnkr.co/edit/MRN9blKg6A3w8G0RkyJD?p=preview)
+
+```html
+<dom-module id="x-foo">
+    <template>
+        <h1><slot name='heading1'></slot></h1>
+        <p><slot name='para'></slot></p>
+    </template>
+</dom-module>
+
+<x-foo>
+    <div slot="heading1">Heading!</div>
+    <div slot="para">Paragraph text.</div>
+</x-foo>
+```
+
+To style slotted content, use the `::slotted()` syntax.
+
+`::slotted(*)` selects all slotted content:
+
+[See it on Plunker](http://plnkr.co/edit/cGrbntuantGtqj7Poqft?p=preview)
+
+```html
+<dom-module id="x-foo">
+    <template>
+      <style>
+        ::slotted(*) {
+          font-family: sans-serif;
+          color:green;
+        }
+      </style>
+        <h1>
+            <div><slot name='heading1'></slot></div>
+        </h1>
+        <p>
+            <slot name='para'></slot>
+        </p>
+    </template>
+</dom-module>
+<x-foo>
+    <div slot="heading1">Heading 1. I'm green.</div>
+    <div slot="para">Paragraph text. I'm green too.</div>
+</x-foo>   
+```
+
+[See it on Plunkr](http://plnkr.co/edit/suVhRDAB5Rq7Q7T6zbMI?p=info)
+
+You can select by element type:
+
+```html
+<dom-module id="x-foo">
+	<template>
```

Comment:
check indentation... I'm going to stop marking these, I think just check the rest.